ports/overgun/gameinfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./OverGun.sh</path>
+    <name>OverGun</name>
+    <desc>OverGun is a shoot'em up 2D platform where your character’s weapon is bigger than he can handle! Take control of the crazy pixeled shooter to help him keep control of his gun and bullets through the dungeon.</desc>
+    <releasedate>20200712T000000</releasedate>
+    <developer>YOUSFI M WALDI</developer>
+    <publisher>YOUSFI M WALDI</publisher>
+    <genre>platformer/strategy</genre>
+    <image>./overGun/cover.png</image>
```

Comment:
<image> tag folderpath is wrong. Needs to match the folder (linux is case sensitive)